hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/supervisor/process/RunningConsumerProcesses.java

```diff
@@ -49,10 +43,12 @@ boolean hasProcess(SubscriptionName subscriptionName) {
         return processes.keySet();
     }
 
-    public List<String> listRunningSubscriptions() {
-        return processes.keySet().stream()
-                .map(SubscriptionName::getQualifiedName)
-                .sorted(String.CASE_INSENSITIVE_ORDER)
+    public List<RunningSubscriptionStatus> listRunningSubscriptions() {
+        return processes.entrySet().stream()
+                .map(entry -> new RunningSubscriptionStatus(
+                        entry.getKey().getQualifiedName(),
+                        entry.getValue().process.getSignalTimesheet()))
```

Comment:
\[FindBugs\] INFO: NP: Read of unwritten field process in pl\.allegro\.tech\.hermes\.consumers\.supervisor\.process\.RunningConsumerProcesses\.lambda$listRunningSubscriptions$1\(Map$Entry\)